README.md

```diff
@@ -227,7 +227,13 @@ Future work and Limitations
 Change Log (From version 2.2.0 and onwards)
 ==============
 
-### __2.6.0-SNAPSHOT__
+### __3.0.0__
+
+* Upgraded to Hive 1.2.1 (Note: new major release with backwards incompatibility issues). As of Hive 1.2 there are a number of new reserved keywords, see [DDL manual](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Keywords,Non-reservedKeywordsandReservedKeywords) for more information. 
+If you happen to have one of these as an identifier, you could either backtick quote them (e.g. \`date\`, \`timestamp\` or \`update\`) or set hive.support.sql11.reserved.keywords=false.                                            
+* Removed the custom HiveConf hive.vs. Use hadoop.tmp.dir instead.   
```

Comment:
So far I can see three different solutions to this issue:

1) Build HiveRunner as one artifact using one version of hive, but let users set a predefined property, for instance 'hive.version', that is used when resolving the HiveRunner dependencies. Unfortunately you need to set this as a system property on the command line when you run mvn (e.g. mvn test .Dhive.version=0.14.0). Specifying it as a property or system property in the user POM (user = the project using HiveRunner, not the HiveRunner pom) is not working.

2) Build two separate artifacts using a multi module project in maven. Since we want to avoid duplicate code, we need to make (at least) three sub modules; the 'main' artifact including the src and test code, two sub modules with just a pom declaring different hive version. With this setup, we need to share the test code (from the 'main' module) in the sub modules. This setup requires a lot of maven code and, depending on how you share the test code, either some serious maven magic or a lot of ugly code in HiveRunner.

3) Git branches. Let git master use the 'latest' dependencies (like hive 1.2 and tez 0.7) and then, when needed, create a specific release branch for hive-0.14. 

Since we haven't been able to find any other suitable solution we would like (for now) to go forward with using git branches (3). This is somewhat unfortunately, but the diff is rather small so keeping the branches in sync should be do-able.